guides/source/active_record_migrations.md

```diff
@@ -357,23 +357,24 @@ will append `ENGINE=BLACKHOLE` to the SQL statement used to create the table
 
 ### Creating a Join Table
 
-Migration method `create_join_table` creates an HABTM join table. A typical use
-would be:
+The migration method `create_join_table` creates an HABTM
+(has_and_belongs_to_many) join table. A typical use would be:
 
 ```ruby
 create_join_table :products, :categories
 ```
 
 which creates a `categories_products` table with two columns called
-`category_id` and `product_id`. These columns have the option `:null` set to
-`false` by default. This can be overridden by specifying the `:column_options`
-option.
+`category_id` and `product_id`. (Note that the table's name is defined by the
+lexical order of the class names - and not by the order of the parameters.)
```

Comment:
Hello! :grimacing: 

I think you can remove this note from this sentence and add it below.